sdk/metric/exemplar/fixed_size_reservoir.go

```diff
@@ -44,18 +44,11 @@ type FixedSizeReservoir struct {
 	// w is the largest random number in a distribution that is used to compute
 	// the next next.
 	w float64
-
-	// rng is used to make sampling decisions.
-	//
-	// Do not use crypto/rand. There is no reason for the decrease in performance
-	// given this is not a security sensitive decision.
-	rng *rand.Rand
 }
 
 func newFixedSizeReservoir(s *storage) *FixedSizeReservoir {
 	r := &FixedSizeReservoir{
 		storage: s,
-		rng:     rand.New(rand.NewSource(time.Now().UnixNano())),
```

Comment:
Have you considered using a global `rng` (from "math/rand"`) instead of creating a new one for each `FixedSizeReservoir`?

I mean I think even

```
var rng *rand.Rand

func init() {
	rng = rand.New(rand.NewSource(time.Now().UnixNano()))
}

Reply:
I see that there was already a similar proposal here: https://github.com/open-telemetry/opentelemetry-go/issues/6260#issuecomment-2639048708